homeassistant/components/homeworks/button.py

```diff
@@ -0,0 +1,78 @@
+"""Support for Lutron Homeworks buttons."""
+from __future__ import annotations
+
+from time import sleep
+
+from pyhomeworks.pyhomeworks import Homeworks
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import HomeworksData, HomeworksEntity
+from .const import (
+    CONF_ADDR,
+    CONF_BUTTONS,
+    CONF_CONTROLLER_ID,
+    CONF_KEYPADS,
+    CONF_NUMBER,
+    CONF_RELEASE_DELAY,
+    DOMAIN,
+)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up Homeworks buttons."""
+    data: HomeworksData = hass.data[DOMAIN][entry.entry_id]
+    controller = data.controller
+    controller_id = entry.options[CONF_CONTROLLER_ID]
+    devs = []
+    for keypad in entry.options.get(CONF_KEYPADS, []):
+        for button in keypad[CONF_BUTTONS]:
+            dev = HomeworksButton(
```

Comment:
Please use variable names names entity etc for entity instances.

Reply:
Fixed in https://github.com/home-assistant/core/pull/114867